bot/tests/test_remote.py

```diff
@@ -250,16 +250,16 @@ def test_decision_task(mock_config, mock_revision, mock_workflow, mock_backend):
     """
 
     mock_workflow.setup_mock_tasks({"decision": {}, "remoteTryTask": {}})
```

Comment:
We no longer have any test for the `Missing decision task` case, could you add it?

Reply:
Done